pkg/kf/commands/install/kf/kf.go

```diff
@@ -176,3 +194,82 @@ func installServiceCatalog(ctx context.Context) error {
 
 	return nil
 }
+
+// fetchReleases returns a list of releases YAML files from github and the
+// nightly. The nightly will be the first entry. The two slices are related.
+func fetchReleases(ctx context.Context) (names, addrs []string, err error) {
+	Logf(ctx, "fetching Kf releases...")
+	versions := []semver.Version{}
+	client := github.NewClient(nil)
+	releases, _, err := client.Repositories.ListReleases(ctx, "google", "kf", nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	for _, r := range releases {
+		var name, tag, assetURL string
+		if r.Name != nil {
+			name = *r.Name
+		}
+
+		if r.TagName != nil {
+			tag = *r.TagName
+		}
+
+		for _, a := range r.Assets {
+			if a.Name != nil && a.BrowserDownloadURL != nil && *a.Name == "release.yaml" {
+				assetURL = *a.BrowserDownloadURL
+				break
+			}
+		}
+
+		if name == "" || assetURL == "" || tag == "" {
+			continue
+		}
+
+		if !strings.HasPrefix(tag, "v") {
+			Logf(ctx, "skipping tag %q, it doesn't have a 'v' prefix", tag)
+			continue
+		}
+
+		// Strip off 'v' prefix
+		version, err := semver.Parse(tag[1:])
+		if err != nil {
+			Logf(ctx, "invalid semver tag %q: %s", tag, err)
+		}
+
+		names = append(names, name)
+		addrs = append(addrs, assetURL)
+		versions = append(versions, version)
+	}
+
+	semver.Sort(versions)
+
+	// We only want the last 3 releases
```

Comment:
I don't think we should limit this until it becomes an issue and we can start to understand how users apply different versions. If we just choose the last N we could end up just picking up patch versions or the like which isn't a very meaningful choice. If we must limit, maybe a time based, I think we could filter releases to only those in the last year.